### ISSUE TITLE
Doubt of use

### ISSUE THREAD

**Issue by JuanCrg90** [jamesfebin/azure-blob-upload#2]
Hi I'm trying to test your package in an application I'm developing, but i have the following problem:

When I upload the image, i obtain in the server, the URL where this is stored, but in the client side i obtain an object.

my question is, is there any way to get the url of the image on the client side, or as would be an appropriate way to use this information I receive. i will use this image with another information in a form

regards.

**Comment by jamesfebin** [comment]
@JuanCrg90 
I haven't added the functionality to receive the url in the client side. So this is how I used it. I used it to update profile pictures of users. I used to update the profile of the user in the server side after the success of the image upload on server side. 

Meteor.user() is reactive. So when a variable changes in the server side it pushes to the client. That's how i received the url in the client side.

That worked for my use case. If you still need to get it at client side you will need to modify the library. I am little busy with work otherwise i would have implemented it.
